content/en/docs/reference/command-line-tools-reference/feature-gates/local-storage-capacity-isolation-fs-quota-monitoring.md

```diff
@@ -9,11 +9,14 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.15"
+  - stage: beta 
+    defaultValue: true
+    fromVersion: "1.31"
 ---
 When `LocalStorageCapacityIsolation`
 is enabled for
-[local ephemeral storage](/docs/concepts/configuration/manage-resources-containers/)
-and the backing filesystem for [emptyDir volumes](/docs/concepts/storage/volumes/#emptydir)
-supports project quotas and they are enabled, use project quotas to monitor
+[local ephemeral storage](/docs/concepts/configuration/manage-resources-containers/),
+the backing filesystem for [emptyDir volumes](/docs/concepts/storage/volumes/#emptydir)
+supports project quotas and they are enabled and when `UserNamespacesSupport` is enabled, use project quotas to monitor
```

Comment:
This sentence doesn't flow ... ?

Reply:
Tried rephrasing it. Please take a look.